readthedocs/settings/base.py

```diff
@@ -66,6 +66,7 @@ class CommunityBaseSettings(Settings):
     DEFAULT_FROM_EMAIL = 'no-reply@readthedocs.org'
     SERVER_EMAIL = DEFAULT_FROM_EMAIL
     SUPPORT_EMAIL = None
+    SUPPORT_FROM_EMAIL = None
```

Comment:
We need to change this one on our ops repos, we can also just use `SUPPORT_EMAIL` I guess.